webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -0,0 +1,260 @@
+                }),
+            }}
+        >
+            <SignIcon />
+            {item.milestone.title}
+        </span>
+      ) : null;
+    
+    let labels: JSX.Element[] | null = item.labels_with_details ? getGitlabLabels(item.labels_with_details) : null;
+
+    let hasConflict: JSX.Element | null = null;
+    if (item.has_conflicts) {
+        hasConflict = (
+            <OverlayTrigger
+                key="gitlabRHSPRMergeableIndicator"
+                placement="top"
+                overlay={
+                    <Tooltip id="gitlabRHSPRMergeableTooltip">
+                        {
+                            "This merge request has conflicts that must be resolved"
+                        }
+                    </Tooltip>
+                }
+            >
+                <i
+                    style={style.conflictIcon}
+                    className="icon icon-alert-outline"
+                />
+            </OverlayTrigger>
+        );
+    }
+
+    let status: JSX.Element | null = null;
+    if (item.status) {
+        switch (item.status) {
+            case "success":
+                status = (
+                    <span
+                        style={{ ...style.icon, ...style.iconSuccess }}
+                    >
+                        <TickIcon />
+                    </span>
+                );
+                break;
+            case "pending":
+                status = (
+                    <span
+                        style={{ ...style.icon, ...style.iconPending }}
+                    >
+                        <DotIcon />
+                    </span>
+                );
+                break;
+            default:
+                status = (
+                    <span
+                        style={{ ...style.icon, ...style.iconFailed }}
+                    >
+                        <CrossIcon />
+                    </span>
+                );
+        }
+    }
+
+    let reviews: JSX.Element | null = null;
+    if(item.total_reviewers && item.approvers){
+        reviews = (
+          <div style={style.subtitle}>
+              <span className="light">
+                  {`${item.approvers} out of ${item.total_reviewers} ${(item.total_reviewers>1 ? "reviews" : "review")} complete.`}
+              </span>
+          </div>
+        )
+    }
+
+    return (
+      <div key={item.id} style={style.container}>
+        <div>
+          <strong>
+              {title}
+              {status}
+              {hasConflict}
+          </strong>
+        </div>
+        <div>
+          {number}
+          <span className='light'>{`(${repoName})`}</span>
+        </div>
+        {labels}
+        <div className='light' style={style.subtitle}>
+          {item.created_at && `Opened ${formatTimeSince(item.created_at)} ago ${userName && ` by ${userName}.`}`}
+          {milestone}
+        </div>
+        <div className="light" style={style.subtitle}>
+        {item.action_name ? (
```

Comment:
```suggestion
        {item.action_name && (
```
and you can omit returning the null on the else condition